src/components/Dashboard/Dashboard.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import NotesContainer from "../NotesContainer/NotesContainer";
```

Comment:
Separate please imports by their type (first rule: https://github.com/k-krasilnikova/traineeship/wiki/React-main-rules), setup absolute paths!

We can't name Container in import, we should use index file also
```suggestion
import Notes from "pages/Notes";
```
and structure will be:
```
src/pages/Notes - folder
src/pages/Notes/index.js - common file for importing container and exporting it
src/pages/Notes/NotesContainer.js - container
src/pages/Notes/Notes.js - component
```